client/components/data/domain-management/index.jsx

```diff
@@ -63,12 +63,12 @@ class DomainManagementData extends React.Component {
 				<CalypsoShoppingCartProvider>
 					{ React.createElement( this.props.component, {
 						context: this.props.context,
-						domains: this.props.selectedSite ? this.props.domains : null,
+						domains: selectedSite ? this.props.domains : null,
 						hasSiteDomainsLoaded: this.props.hasSiteDomainsLoaded,
 						isRequestingSiteDomains: this.props.isRequestingSiteDomains,
 						products: this.props.products,
 						selectedDomainName: this.props.selectedDomainName,
-						selectedSite: this.props.selectedSite,
+						selectedSite: selectedSite,
```

Comment:
nit: just my personal preference, but we could shorten this to just `selectedSite,` – also, is there a reason we only destructure certain props and not all? don't feel obligued to change this, it's purely stylistic.

Reply:
> nit: just my personal preference, but we could shorten this to just selectedSite,

Done, good call!

> also, is there a reason we only destructure certain props and not all? don't feel obligued to change this, it's purely stylistic.

No reason 😉  Just trying to limit the number of changes I guess.